molgenis-data-excel/src/main/java/org/molgenis/data/excel/ExcelEntity.java

```diff
@@ -23,6 +24,8 @@
 	private final Map<String, Integer> colNamesMap;
 	private final List<CellProcessor> cellProcessors;
 
+	private transient Map<String, Object> cachedValueMap;
```

Comment:
> Why are not all class parameters final transient?
Entity is serializable: Writing it to disk and reading it back into memory requires that the other fields still have values in order for the entity to work as before serialization. Row should not be transient by the way, but that code is already in place.

> cachedValueMap can this not be already instantiated?
Lazy initialization pattern to improve performance of ExcelEntity in case you create the entity but e.g. skip over it when iterating entities.